hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenGrantKycToAccountHandlerTest.java

```diff
@@ -139,7 +141,7 @@ private ReadableTokenStore mockKnownKycTokenStore() {
                 false,
                 TokenType.FUNGIBLE_COMMON,
                 TokenSupplyType.INFINITE,
-                -1,
+                AccountID.newBuilder().accountNum(-1).build(),
```

Comment:
Should be null. There is no valid account with negative numbers

Reply:
Updated, thanks.